presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -1638,4 +1640,17 @@ public Duration getPartitionLeaseDuration()
     {
         return partitionLeaseDuration;
     }
+
+    public boolean isExecutionBasedMemoryAccounting()
+    {
+        return executionBasedMemoryAccounting;
+    }
+
+    @Config("hive.execution-based-memory-accounting")
+    @ConfigDescription("When enabled sets the memory accounting flags for the orc writer. This is enabled for Presto on Spark.")
```

Comment:
Maybe something like "When enabled this removes some memory accounting that can cause queries in existing workloads to hit memory limits they previously fit in."